### PR TITLE
Bug 1132914 - Change message to be more clear that speed dial loads cont...

### PR DESCRIPTION
**Pull request by tamarahills**
...acts from SIM. r=drs.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2167391501a0)

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dfd0b2eaaa40)

File: shared/js/dialer/keypad.js [hunk 1]
```diff
@@ -635,7 +635,7 @@ var KeypadManager = {
             reject();
           };
 
-          ConfirmDialog.show('loadingContacts', null, {
+          ConfirmDialog.show('loadingSimContacts', null, {
             title: 'cancel',
```
Comment (DouglasSherk) [inline shared/js/dialer/keypad.js]: This should be tested in ```apps/communications/dialer/test/unit/keypad_test.js```. It should be a fairly trivial change, just to make it more specific about what's being passed in.

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bc56389d06cf)

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4462d8c7fe78)

**Pull request merged**